src/lesson2/HomeWork2.java

```diff
@@ -0,0 +1,49 @@
+package lesson2;
+
+public class HomeWork2 {
+
+  
+
+    public static void main(String[] args) {
+        System.out.println(summ(15, 8));
+        isPositiveOrNegative(8);
+        System.out.println(isNegative(19));
+        printWordNTimes();
+        LeapYear(1996);
+    }
+
+
+
+    public static boolean summ(int a, int b) {
+        int sum = a + b;
+        if (sum > 10 && sum < 20) return true;
+        else return false;
```

Comment:
можно упростить, поскольку логическое выражение само по себе является boolean значением на выходе:
```suggestion
        return sum > 10 && sum < 20;
```